module/zfs/zvol.c

```diff
@@ -2010,7 +2010,7 @@ zvol_remove_minors_impl(const char *name)
 static void
 zvol_remove_minor_impl(const char *name)
 {
-	zvol_state_t *zv, *zv_next;
+	zvol_state_t *zv = NULL, *zv_next;
```

Comment:
It is a matter of style, I suppose, I would not initialize the zv explicitly then.